server/src/db/schema.ts

```diff
@@ -109,7 +109,7 @@ export const template = sqliteTable("template", {
 	createdBy: text("created_by"),
 	createdAt: integer('created_at', { mode: 'timestamp' }).default(sql`(strftime('%s', 'now'))`),
 	updatedAt: integer('created_at', { mode: 'timestamp' }),
-	createdByDocument: text('user_id)').references(()=> user.id),
+	createdByDocument: text('user_id)').references(()=> UserTable.id),
```

Comment:
All of these 'document' columns we had in prisma should probably just go back to how we had it in mongoose, like this would be createdBy not createdByDocument